docs/instruments/instrument_sql/Meta/mri_parameter_form_meta.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO test_names (Test_name, Full_name, Sub_group) VALUES('mri_parameter_form', 'MRI Parameter Form', 2);
```

Comment:
SELECT 'medical_history', 'Medical History', ID FROM test_subgroups WHERE Subgroup_name = 'Imaging';

Reply:
should I put `mri_parameter_form` instead here ? otherwise it gets confusing

Reply:
`mri_parameter_form` is an instrument.